rosbag2_performance/rosbag2_performance_benchmarking/config/producers/automotive.yaml

```diff
@@ -34,18 +31,18 @@ rosbag2_performance_benchmarking_node:
       gps_5KBps:
         publishers_count:   1
         topic_root:         "gps"
-        msg_type:           "sensor_msgs::msg::NavSatFix"
+        msg_size_bytes:     200
         msg_count_each:     250
         rate_hz:            25
       imu_40KBps:
         publishers_count:   1
         topic_root:         "imu"
-        msg_type:           "sensor_msgs::msg::Imu"
+        msg_size_bytes:     400
         msg_count_each:     1000
         rate_hz:            100
       ultrasonic_72KBps:
         publishers_count:   12
         topic_root:         "ultrasonic"
-        msg_type:           "sensor_msgs::msg::Range"
+        msg_size_bytes:     150
```

Comment:
@MichaelOrlov I would add an additional benchmark using the `msg_type` option in order to verify it is working. We can create two versions of the automotive YAML files, one with message types and another with generic types and message sizes. If we don't want to extend the benchmark time we can create a simpler benchmark with `msg_type` just to cover this case, i.e. 1 topic, 100 messages.

Reply:
@carlossvg Good catch I missed this point.
I added dedicated non default  `msg_type` for camera images and point clouds in automotive.yaml config file.
For other types we don't have type specialized `resize()` method.